Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Calibration/CalculatedCameraExtrinsics.cs

```diff
@@ -22,5 +24,28 @@ public override string ToString()
         {
             return $"Succeeded: {Succeeded} {base.ToString()}";
         }
+
+        public byte[] Serialize()
+        {
+            var str = JsonUtility.ToJson(this);
+            var payload = Encoding.ASCII.GetBytes(str);
+            return payload;
+        }
+
+        public static bool TryDeserialize(byte[] payload, out CalculatedCameraExtrinsics extrinsics)
+        {
+            extrinsics = null;
+
+            try
+            {
+                var str = Encoding.ASCII.GetString(payload);
+                extrinsics = JsonUtility.FromJson<CalculatedCameraExtrinsics>(str);
+                return true;
+            }
+            catch
```

Comment:
It might be worth logging the exception (in case the JSON library has some helpful message about what part failed deserialization)

Reply:
i'll add this to both extrinsics/intrinsics